src/appleseed-max-impl/appleseedlightmtl/appleseedlightmtl.cpp

```diff
@@ -335,6 +342,9 @@ void AppleseedLightMtl::Update(TimeValue t, Interval& valid)
         m_pblock->GetValue(ParamIdLightColorTexmap, t, m_light_color_texmap, m_params_validity);
 
         m_pblock->GetValue(ParamIdLightPower, t, m_light_power, m_params_validity);
+        float exposure;
+        m_pblock->GetValue(ParamIdLightExposure, t, exposure, m_params_validity);
+        m_light_power *= std::pow(2.0f, exposure);
```

Comment:
You need to include `<cmath>`.

Reply:
Also, side note: the `DiffuseEDF` model actually has an `exposure` parameter. The upside of using it would be that it can be textured, but since we don't seem to allow this in this material, the way you did it is fine.

Reply:
Actually, now that I think of it, it would be nicer to use the `exposure` parameter natively supported by the EDF so that it appears as such when the scenes are opened in appleseed.studio. Can be changed in a separate PR.